components/director/internal/domain/application/resolver.go

```diff
@@ -264,6 +264,9 @@ func (r *Resolver) RegisterApplication(ctx context.Context, in graphql.Applicati
 	log.C(ctx).Infof("Registering Application with name %s", in.Name)
 
 	convertedIn, err := r.appConverter.CreateInputFromGraphQL(ctx, in)
+
+	convertedIn.Labels["managed"] = false
```

Comment:
Maybe do this after check for err? I personally got confused while reading this part and thought that we swallow an error on line 266.